src/config/texts.js

```diff
@@ -925,6 +925,13 @@ export const texts = {
         imageGreater10MBError: 'Das ausgewählte Bild darf maximal 10 MB groß sein.',
         imagesGreater30MBError:
           'Die ausgewählten Bilder dürfen insgesamt nicht größer als 30 MB sein.',
+        imageSelectAlert: {
+          camera: 'Kamera',
+          cancel: 'Abbrechen',
+          description: 'Möchten Sie das Bild aus der Kamera oder aus der Galerie auswählen?',
```

Comment:
please change to: Möchten Sie ein Bild mit der Kamera aufnehmen oder aus der Galerie auswählen?

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/482c4d81085f4d491efb9b2067efc4929b96b4cf